src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -529,12 +529,21 @@ public CradleManager getCradleManager() {
      */
     public abstract InputStream loadSingleDictionary();
 
+    /**
+     * @param alias name of dictionary
+     * @return Dictionary as {@link File}
+     * @throws IllegalStateException if can not read dictionary
+     */
+    public abstract File loadDictionaryFile(String alias);
```

Comment:
I think the name should be `getDicrionaryFile` or `getPathToDictionary`. And it is better to use `Path` as a result I think